docs/source/upgrade-guide/index.md

```diff
@@ -46,6 +46,24 @@ There's nothing that limits to move all the config from a project to an add-on p
 See {ref}`upgrade-18-cookieplone-label` for details.
 
 
+### Plone 5 is no longer tested
+```{versionremoved} Volto 19
+```
+
+Volto 19 no longer includes automated tests for compatibility with Plone 5. While it may still work with Plone 5 backends in some cases, we recommend upgrading to Plone 6 for full compatibility and support.
```

Comment:
```suggestion
Volto 19 no longer includes automated tests for compatibility with Plone 5.
While it may still work with Plone 5 backends in some cases, we recommend upgrading to Plone 6 for full compatibility and support.
```